### ISSUE TITLE
import and babel-runtime/helpers/classCallCheck not found

### ISSUE THREAD

**Issue by lucasriondel** [zeroturnaround/sql-formatter#16]
shouldn't you use require to import the script ? 
when i try to run it, it tells me

```
Error: Cannot find module 'babel-runtime/helpers/classCallCheck'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/lucas/dev/fivegoal_reservationsapi/node_modules/sql-formatter/lib/languages/N1qlFormatter.js:5:24)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/lucas/dev/fivegoal_reservationsapi/node_modules/sql-formatter/lib/sqlFormatter.js:5:22)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
```

**Comment by nene** [comment]
Looks like we're missing `babel-runtime` in our dependencies. I thought that by using `babel-plugin-transform-runtime` we don't need to add `babel-runtime` itself as dependency, but we do. From [the docs](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-runtime#technical-details):

> Make sure you include babel-runtime as a dependency.

**Comment by ukupat** [comment]
Fixed in v1.1.1 https://github.com/zeroturnaround/sql-formatter/releases/tag/v1.1.1 @lucasriondel 

Thanks for reporting!
